2_munge.yml

```diff
@@ -13,6 +13,7 @@ targets:
   2_munge:
     depends:
       - 1_data/out/nwis.rds.ind
```

Comment:
i think we should delete line 15 now - was probably just a placeholder

Reply:
yup